cocos/physics/CCPhysicsWorld.h

```diff
@@ -358,6 +366,8 @@ class CC_DLL PhysicsWorld
     float _updateTime;
     int _substeps;
     cpSpace* _cpSpace;
+    float _updateFilteredDelta;
```

Comment:
Could we remove the member variable `_updateFilteredDelta` and leave it a local variable in the `world::step` method?